tests/console/consoletest_setup.pm

```diff
@@ -8,8 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Rework the tests layout.
-# G-Maintainer: Alberto Planas <aplanas@suse.com>
+# Summary: Prepare console for console tests
+# Maintainer: Alberto Planas <aplanas@suse.com>
```

Comment:
Please put someone reasonable, e.g. you, me, coolo, any person that has been actively looking into this recently